packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -350,12 +350,12 @@ void main() {
 
       test('file with special characters', () async {
         final VideoPlayerController controller =
-            VideoPlayerController.file(File('A #1 Hit?.avi'));
+            VideoPlayerController.file(File('A #1 Hit.avi'));
```

Comment:
It turns out `?` is an illegal character in a Windows filename, so this test didn't work on Windows. The test still validates URL escaping without the ? so I just removed it rather than conditionalize by platform.